ooni/templates/process.py

```diff
@@ -103,12 +103,10 @@ def _setUp(self):
 
     def processEnded(self, result, command):
         log.debug("Finished %s: %s" % (command, result))
-        key = ' '.join(command)
-        self.report[key] = {
-            'stdout': result['stdout'],
-            'stderr': result['stderr'],
-            'exit_reason': result['exit_reason']
-        }
+        self.report['command_name'] = ' '.join(command)
```

Comment:
Do you mean a list of dictionaries?, could you write an example output where there would be several several commands, each of them with a list of the key-values you mentioned in https://github.com/TheTorProject/ooni-probe/pull/445#issuecomment-163318796?
Would it be something like:
- { command_exit_reason: process_done, command_name: /home/foo/.virtualenvs/ooniprobedev/bin/python /tmp/tmpXBR39b }
- { command_exit_reason: connection_refused, command_name: /home/foo/.virtualenvs/ooniprobedev/bin/python /tmp/tmpfoo }